src/components/Markdown/markdown.scss

```diff
@@ -21,6 +21,7 @@
 .page-p--md {
   //md default
   @include type-style('body-short-02');
+  margin-bottom: 0.5rem;
```

Comment:
Are we sure this doesn't throw off spacing anywhere else in the site?

Reply:
When I searched the project to see what where `<p size="md">` was being used, it was just in these subpages for these bottom cards. So I don't think it should, unless markdown paragraphs are using this class inherently?

Reply:
Same for the h5. Looked at both `h5` and `#####`.

Reply:
size md is the default for any markdown paragraph that isn't set.

Reply:
Ah. I thought the default was the `page-p` class. I can go change the spacing back if need be. I just went and inspected the p's to double check and they all had the `page-p` class and these were the only ones that had `page-p--md`.

Reply:
oh ok. ignore me then 😉

Reply:
lol no worries. Doesn't hurt to double check 👍